Makefile

```diff
@@ -123,6 +123,7 @@ devtools-stamp: tools
 	@echo "--> Downloading linters (this may take awhile)"
 	$(GOPATH)/src/github.com/alecthomas/gometalinter/scripts/install.sh -b $(GOBIN)
 	go get github.com/tendermint/lint/golint
+	go install ./cmd/sdkch
```

Comment:
we should be able to execute a workable version (maybe one that manually asks for each input such as the section and stanza, name and contents) from a command like `make cl`. A new developer should not be required to understand `sdkch` it should just be easy and intuitive when this run `make cl`

Reply:
I don't get it, what should `make cl` do? Compile sdkch or execute it?

Reply:
I don't understand this. What should `make cl` do? `sdkch` is supposed to be a developer tool, I fail to grasp the use case of running it from `Make`.

Reply:
I have to agree here with @alessio -- there is no benefit of adding a command to make where you'll just have to end up providing all the input anyway.

Reply:
@rigelrozanski are you perhaps asking for some sort of interactive mode? I mean, what are the expections on new developers? I suppose it's fair to assume they would be able to launch `sdkch -help` and quickly learn its usage.

However, since all comments are addressed are three +1 have been given, can we get this over the line ASAP please? cc @cwgoes

Reply:
k leave for now